web_responsive/static/src/less/navbar.less

```diff
@@ -3,6 +3,7 @@
 
 .main-nav {
     margin-bottom: 0;
+    z-index: 9000;
```

Comment:
With this we are hiding some elements behind (like the notification bar for adding in the browser)